src/operations/Completion.cpp

```diff
@@ -199,73 +199,169 @@ static std::vector<std::string> getServiceNames(const Luau::ScopePtr& scope)
     return services;
 }
 
+static lsp::CompletionItem createSuggestService(const std::string& service, size_t lineNumber)
+{
+    auto importText = "local " + service + " = game:GetService(\"" + service + "\")\n";
```

Comment:
Use the configured line endings for the workspace instead of the hardcoded end of line here?

Reply:
Sure, but I think this should be a separate PR. I was just doing some refactoring and moved it out of the for loop.

Reply:
Adding on to this, there's hard coded line endings everywhere in the code base and is therefore not related to this PR.